File: Typing/typing.py
import time
text = "My name is Oyinade"
print(text)

start = time.time()
user_input = input("Enter the text above :  ")	
end = time.time()
total_time  = end - start
total_seconds = total_time;
		
print(f"The total time taken is {total_seconds} seconds")
number_of_words = len(text.split(" "));
current_minutes =  total_seconds
word_per_minutes = number_of_words / current_minutes;

print(f"The word per minutes  {word_per_minutes}");	

	

File: Typing/Typing.js

const prompt = require('prompt-sync')();


const text = "My name is Oyinade";
	console.log(text);

const start = new Date();
	let input = prompt("Enter the text above :  ");	
const end = new Date();
		let totalTime  = end - start ;
		let  totalSeconds = totalTime / 1000;
		
		console.log("The total time taken is" + totalSeconds + " seconds");

		let numberOfWords = text.split(" ").length;
		let currentMinutes =  totalSeconds/ 60.0;
		let wordPerMinutes = numberOfWords / currentMinutes;

		console.log("The word per minutes " + wordPerMinutes);	

	